slices/libavcodec60.yaml

```diff
@@ -0,0 +1,52 @@
+package: libavcodec60
+
+essential:
+  - libavcodec60_copyright
+
+slices:
+  libs:
+    essential:
+      - libaom3_libs
+      - libavutil58_libs
+      - libc6_libs
+      - libcairo2_libs
+      - libcodec2-1.2_libs
+      - libdav1d7_libs
+      - libglib2.0-0t64_libs
+      - libgsm1_libs
+      - libjxl0.7_libs
+      - liblzma5_libs
+      - libmp3lame0_libs
+      - libopenjp2-7_libs
+      - libopus0_libs
+      # The librav1e0_libs dependency is currently ommited, it can be added again
+      # once chisel supports per-arch package dependencies,
+      # see https://github.com/canonical/chisel/issues/93
```

Comment:
This is a nice comment, thank you. Would you kindly also include which architecture(s) requires the slice?

Reply:
Updated in 07333dc